packages/node-build-scripts/package.json

```diff
@@ -56,7 +56,7 @@
         "jest": "^29.7.0"
     },
     "engines": {
-        "node": ">=20.11"
+        "node": ">=24.11"
```

Comment:
Is upgrading the engines filed in `@blueprintjs/webpack-build-scripts` and `@blueprintjs/node-build-scripts` a breaking API change for those packages?

Reply:
Ah, yeah, looks like it should be based on https://github.com/palantir/blueprint/pull/6661